### DISCUSSION TITLE
Frage: RF-Band nach Hoymiles scannen?

### DISCUSSION THREAD

**Discussion by spilles** [tbnobody/OpenDTU#1522] [General]
Hallo zusammen!

Gibt es eine Möglichkeit, das 2,4Ghz-Band des Radio-Moduls nach Wechselrichtern zu scannen? Z.B über einen Konsolenbefehl oder so...

Hintergrund: Ich hab einen MI-1200 auf dem Dach, aber grad keine Seriennummer zur Hand. Wollte erst nur mal sehen, ob er HF-technisch zu erreichen wäre.

Danke und Gruß: Spilles

**Comment by stefan123t** [comment]
Hallo @spilles nein das Search / Gongfa Command hat bisher noch keiner getraced bzw. implementiert.
Es soll das wohl geben aber wir haben den Inverter bisher noch nicht dazu bekommen sich einfach ohne seine Serial ID zu melden.

Ich weiß auch nicht ob die Gen2 MI-1200 Inverter von OpenDTU überhaupt unterstützt werden ?
Vermutlich hast Du mehr Erfolg mit den Code Anpassungen von rejoe2, u.a. beim Schwesterprojekt Ahoy DTU, dort sollten MI-1200 Inverter abgefragt werden können.
